docs_source_files/content/guidelines_and_recommendations/consider_using_a_fluent_api.en.md

```diff
@@ -3,8 +3,8 @@ title: "Consider using a fluent API"
 weight: 8
 ---
 
-Martin Fowler coined the term "Fluent API". Selenium already
-implements something like this in their `FluentWait` class which is
+Martin Fowler coined the term ["Fluent API"](//www.martinfowler.com/bliki/FluentInterface.html). Selenium already
```

Comment:
Great! Could you please add the same link to the other translations? To keep them uniform.

Reply:
Updated.